File: docs/components_page/components/list_group/colors.R
library(dashBootstrapComponents)

list_group <- dbcListGroup(
  list(
    dbcListGroupItem("No color applied"),
    dbcListGroupItem("The primary item", color = "primary"),
    dbcListGroupItem("A secondary item", color = "secondary"),
    dbcListGroupItem("A successful item", color = "success"),
    dbcListGroupItem("A warning item", color = "warning"),
    dbcListGroupItem("A dangerous item", color = "danger"),
    dbcListGroupItem("An informative item", color = "info"),
    dbcListGroupItem("A light item", color = "light"),
    dbcListGroupItem("A dark item", color = "dark")
  )
)

File: docs/components_page/components/list_group/colors.jl
using DashBootstrapComponents

list_group = dbc_listgroup([
    dbc_listgroupitem("No color applied"),
    dbc_listgroupitem("The primary item", color = "primary"),
    dbc_listgroupitem("A secondary item", color = "secondary"),
    dbc_listgroupitem("A successful item", color = "success"),
    dbc_listgroupitem("A warning item", color = "warning"),
    dbc_listgroupitem("A dangerous item", color = "danger"),
    dbc_listgroupitem("An informative item", color = "info"),
    dbc_listgroupitem("A light item", color = "light"),
    dbc_listgroupitem("A dark item", color = "dark"),
]);
